config/core/build-configs.yaml

```diff
@@ -966,6 +969,25 @@ build_configs:
     tree: renesas
     branch: 'next'
 
+  riscv-fixes: &riscv-tree
+    tree: riscv
+    branch: 'fixes'
+    variants:
+      gcc-10:
+        build_environment: gcc-10
+        architectures:
+          riscv:
+            base_defconfig: 'defconfig'
+            extra_configs:
+              - 'nommu_k210_defconfig'
+              - 'nommu_k210_sdcard_defconfig'
+              - 'nommu_virt_defconfig'
+              - 'rv32_defconfig'
```

Comment:
Please leave off the `extra_configs` section.  We just want it to build all upstream defconfigs, which it will do by default, so pretty sure this is redundant.

Reply:
Done